### ISSUE TITLE
How to paste multiple zettel links?

### ISSUE THREAD

**Issue by MartyLake** [michal-h21/vim-zettel#151]
When I do `[[`, I get the FZF menu to choose a note and if I press enter it gets inserted.
I wish there was a way to insert many notes at once, by multi-selecting them in FZF.

**Comment by michal-h21** [comment]
Good idea, thanks. I think it should work with the latest update. You can select mutliple notes using <shift><tab> and then insert them using <enter>.

**Comment by MartyLake** [comment]
neat! it works great, thankt!!!

**Comment by michal-h21** [comment]
Thanks for the confirmation, I think I can close this now :)
